src/flake8/formatting/default.py

```diff
@@ -45,11 +54,10 @@ def format(self, error: "Violation") -> Optional[str]:
         file. Otherwise, print the formatted error to standard out.
         """
         return self.error_format % {
-            "code": error.code,
-            "text": error.text,
-            "path": error.filename,
-            "row": error.line_number,
-            "col": error.column_number,
+            **{
+                key: getattr(error, error_attr)
+                for key, error_attr in MAPPING_KEYS.items()
+            },
```

Comment:
this is over-DRY for little-to-no benefit and turns off the type checker for these

Reply:
Yeah, that's unfriendly to the type checker. Addressed in 5f14bc5021a9a1668befbd2933bc6aa33b9ee65b. Let me know if you have any other ideas.